lib/v2/cmpxchg8b/articles.js

```diff
@@ -0,0 +1,44 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const baseUrl = 'https://lock.cmpxchg8b.com/';
+const title = 'cmpxchg8b';
+
+module.exports = async (ctx) => {
+    const { data: response } = await got({
+        method: 'get',
+        url: baseUrl,
+    });
```

Comment:
https://github.com/DIYgod/RSSHub/blob/31774e8b7cd8e5d38e5c4f05b87ba10c6081e9c7/lib/v2/cmpxchg8b/articles.js#L29
Please use either one style but not both.